package.json

```diff
@@ -2,14 +2,19 @@
   "name": "mtrpz-lab-4",
   "version": "1.0.0",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "npx mocha --config ./.mocharc.js"
```

Comment:
Change the extension of the .mocharc.js file from .js to .json:
```
"test": "npx mocha --config ./.mocharc.json"
```

Reply:
Ok.